src/parcel.js

```diff
@@ -41,7 +41,7 @@ export default class Parcel extends React.Component {
   componentDidUpdate() {
     this.addThingToDo('update', () => {
       if (this.parcel && this.parcel.update) {
-        this.parcel.update(this.getCustomProps())
+        return this.parcel.update(this.getCustomProps())
```

Comment:
Getting a test to capture this would be awesome. Although maybe sort of tricky. I think what it would entail is to call `wrapper.update()` multiple times in a row without waiting for the parcel `update()` promise to finish. And then verify that we don't call parcel update until the previous update finishes.

If it turns out to be too hard to get a test to do that, I'm cool with not doing so. But let's try it out at least